client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -17,130 +17,277 @@
 
 package org.apache.spark.shuffle;
 
-import java.util.Set;
-
 import com.google.common.collect.Sets;
+import org.apache.spark.internal.config.ConfigBuilder;
+import org.apache.spark.internal.config.ConfigEntry;
 
-import org.apache.uniffle.client.util.RssClientConfig;
+import java.util.Set;
 
 public class RssSparkConfig {
 
+  public static final class SparkConfigBuilder {
```

Comment:
Could we use ConfigBuilder and ConfigEntry directly? It seems more clear.

Reply:
The reason why I encapsulate it here is to support type conversion. If configbuilder or configintry is used, there will be problems during type conversion.

Reply:
What's the problems?

Reply:
When we use ConfigEntry as the return `object`, the return type is not allowed to be wrapper classes such as Integer and Long. We can only select `ConfigEntry<Object>` or `ConfigEntry<String>`, which will throw an exception when sparkConf parses parameters. Methods like `sparkConf.getLong()` have no way to complete the type conversion from Object to Integer, which will lead to compilation failure.

Reply:
OK